nima/webclient/api/src/main/java/io/helidon/nima/webclient/api/HttpClientConfigBlueprint.java

```diff
@@ -151,7 +151,7 @@ default ClientRequestHeaders defaultRequestHeaders() {
      *
      * @return media context
      */
-    @ConfiguredOption
+    @ConfiguredOption("create()")
```

Comment:
Is "create()" a config key  ?

Reply:
create() is a static method on the type - this is the default value

Reply:
create() is a static method on the type - this is the default value